### ISSUE TITLE
Dugr 4 Beta: Issue creating groups for Duik Structures and Controllers

### ISSUE THREAD

**Issue by popevideo** [RxLaboratory/DuAEF_DuGR#30]
There is an issue creating groups for Duik Structures and Controllers. I believe it may have something to do with way Duik adds markers to the Structure and Controller layer and the new way Dugr adds markers to the group layers.

**Comment by Nico-Duduf** [comment]
I forgot to test this with the current version of Duik, thanks for the feedback!
I'll fix that!

**Comment by popevideo** [comment]
Thank you so much for your hard work! New version of Dugr is looking good and makes using AE such better user experience.

**Comment by popevideo** [comment]
It also appears Pre-Comps will not add to groups as well

**Comment by Nico-Duduf** [comment]
Thanks for the feedback, I'll fix that too

**Comment by Nico-Duduf** [comment]
I can't reproduce any issue here with precomps and Duik layers, can you be more specific about what's going on?
In the settings, you can enable the "dev & debug" mode and copy here any error which may be shown

**Comment by popevideo** [comment]
Ok I've updated AE to version 18.4.0(Build 41) and now Structures and Controllers are working. I tried to group a Pre-Comp but it is still not letting me. I switched to Bug & Dev mode and I got the attached error.  I'm on a Mac using macOS 11.5.1
<img width="259" alt="Screen Shot 2021-08-05 at 12 30 26 PM" src="https://user-images.githubusercontent.com/46870817/128410152-fcc84215-c626-4f3c-b6c2-7e5c3dae50e9.png">
